package-lock.json

```diff
@@ -1507,7 +1639,8 @@
         "ansi-regex": {
           "version": "2.1.1",
           "bundled": true,
-          "dev": true
+          "dev": true,
+          "optional": true
```

Comment:
I am okay with `"optional": true` but I know Inscrutable had some concerns with it in the automatic PR. I point them out to ensure they are not an oversight.

Reply:
Completely autogenerated by npm, so it's fine.